lib/cdo/mailjet.rb

```diff
@@ -0,0 +1,110 @@
+require 'mailjet'
+require 'honeybadger/ruby'
+require_relative './shared_constants/mailjet_constants'
+
+module MailJet
+  include MailJetConstants
+
+  API_KEY = CDO.try(:mailjet_api_key).freeze
+  SECRET_KEY = CDO.try(:mailjet_secret_key).freeze
+
+  def self.enabled?
+    return false unless DCDO.get('use_mailjet', false)
+    !API_KEY.nil? && !SECRET_KEY.nil?
+  end
+
+  def self.subaccount
+    case Rails.env
+    when 'production'
+      'production'
+    when 'staging'
+      'staging'
+    else
+      'development'
+    end
+  end
+
+  def self.create_contact_and_send_welcome_email(user)
+    return unless enabled?
+
+    return if user&.id.nil?
+    return unless user.teacher?
+
+    create_contact(user.email, user.name, user.created_at.to_datetime.rfc3339)
```

Comment:
Can there be a comment describing rfc3339?

Reply:
I'm guessing we're doing this because Mailjet supports rfc3339 formatted dates (iso-8601 would probably be fine too, but they specifically name rfc3339)

I'd prefer we shove the mailjet-specific formatting down into the `create_contact` function, then it's more clear that we're doing this explicitly for Mailjet.

Reply:
Darin, is the change in ade564409238746dbe46c9a771b946eae818c0cc what you were thinking?

Reply:
Yep!